12343 add high res check img

dsrr47 opened the pull request:
## Description
The personal check image is a bit blurry on higher resolution screens. It might be worth considering using a higher resolution image to illustrate the points about routing numbers and account numbers. Screenshot attached below.
<img width="950" alt="89854903-bcd77d80-db5a-11ea-94a9-49415bd092fa" src="https://user-images.githubusercontent.com/18352271/140090432-2d935e0f-9d59-46e7-ab42-b147282111ed.png">


## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/12343

SVG provided 
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/assets/img/direct-deposit-check-guide.svg

## Testing done
local environment


## Screenshots
<img width="1079" alt="Screen Shot 2021-11-03 at 10 45 16 AM" src="https://user-images.githubusercontent.com/18352271/140094185-4409a04a-1316-4ccd-91f4-31a36fa040dc.png">

## Acceptance criteria
- [ ] Check image is higher resolution

## Definition of done
- [ ] Acceptance criteria met
- [ ] Production flags used.
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

Commit:
updated direct deposit check image to higher res image

Commit:
added image to site assets

Commit:
master sync

Commit:
master sync

Commit:
master sync

bkjohnson reviewed (changes requested):
Will you include the SVG file that was used to generate the PNG? Also, I don't see a reason to maintain both the old version and the newer high res version.

dsrr47 commented:
Okay,  updated the PR with the original svg. 

I left the original for the time being because of the environment flags. The original svg will still be used on production until the flag is removed. Once this gets verified for production I'll remove the original png along with the environment flags.

bkjohnson commented:
I was asking for the SVG that was used to generate the high-res PNG to be committed to git. Right now it isn't in version control, which means there isn't an easy way to make further modifications to the PNG in the future.

![image](https://user-images.githubusercontent.com/2008881/140237354-ba4f566a-8044-4277-8f73-5a886138a3e2.png)


Currently in the [`src/site/assets/img` directory](https://github.com/department-of-veterans-affairs/vets-website/tree/102384255107ea007f2d7d03ad4738fcdb11a3a8/src/site/assets/img) we have the original SVG, and a lower resolution PNG which may have been generated from it at some point in the past.

---

Additionally, I asked because this appears to be the PNG that I made as [part of my discovery for the original issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12343#issuecomment-940553362). In that comment I said:

> I didn't spend much time on aligning things properly, but this is an example of what we could do.

That PNG that I made was not meant to be used in production due to the problems I mentioned then and am pointing out now.

The reason this looks like the PNG I made is because if you zoom in closely, you can see how the font is distorted where I stretched some things out. I took screenshots of the PNG from your branch and stacked them vertically - I didn't resize anything.

![image](https://user-images.githubusercontent.com/2008881/140237780-2bc13d3f-ced2-481e-b221-0bb9afb7e53f.png)

Even if this isn't a copy of my draft image, I don't think we should use an image with this kind of distortion.

Commit:
Merge remote-tracking branch 'origin' into 12343-add-high-res-check-img
master sync

Commit:
added svg and modified routing and account numbers to not be an actual account.
generated new png from svg without text distorition and added it to the page.

dsrr47 commented:
@bkjohnson I added the svg to the PR and replaced the old png that had distorted text with one that was generated directly from the svg. Let me know if that png works. If it doesn't can you please provide me one?

erikphansen commented on src/applications/personalization/profile/components/direct-deposit/BankInfo.jsx:
```diff
@@ -287,7 +288,11 @@ export const BankInfo = ({
             {/* eslint-disable jsx-a11y/no-redundant-roles */}
             <img
-              src="/img/direct-deposit-check-guide.svg"
+              src={
```
why replace an SVG with a PNG? I did just notice that for some reason the current SVG does not seem to exist on staging, but it does show up locally. So that's something that needs to be looked into; I swear it used to work since the SVG was approved in staging months ago.

dsrr47 replied:
I used a PNG because in the [original story](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/12343) it was recommended in order to get a higher resolution. I can use the SVG if thats is preferred format.

bkjohnson replied:
From [my comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12343#issuecomment-940526865) on that issue:

> You may be able to get a higher resolution png by creating an export from the svg file.

It doesn't look like there are any comments there recommending using PNGs over SVGs for their resolution. Let me know if I'm overlooking something.

dsrr47 replied:
Okay, is the SVG preferred then?

bkjohnson reviewed (changes requested):
The spacing between some elements in the PNG looks a bit different, I didn't measure though:

![image](https://user-images.githubusercontent.com/2008881/142696009-5e28f2ae-1cd2-457e-8897-aae1852c550a.png)

---

The SVG also doesn't look good to me. Is this what you see?

![image](https://user-images.githubusercontent.com/2008881/142696292-900ba769-4fa6-4305-abea-fee02dafe4f4.png)

Can you describe the steps you took to generate the PNG?

> Let me know if that png works. If it doesn't can you please provide me one?

It doesn't look to me like that PNG was generated from the provided SVG, so right now I'm going to say that it doesn't work. Asking about how to generate PNGs from SVGs would be an appropriate conversation to have in the original issue before PR work begins. In my comment that was edited by someone other than myself (see below), I provided an example PNG that I generated from the SVG which I modified, and I generated the image using Inkscape.

This hasn't been prioritized by my team, so I won't be spending further time modifying the SVG to something that is acceptable and generating a PNG unless that prioritization changes.

---

I also went back to the original issue and it appears that my comment was edited. @mrblanco1, please don't do this in order to misrepresent any examples I'm providing. My example was of a higher resolution check PNG by itself, not the original image on a form page with 0s edited on top of it.

![image](https://user-images.githubusercontent.com/2008881/142697213-e902699b-8c11-4688-9389-6df69f240909.png)

dsrr47 commented:
This is the SVG I see. Its definitely different from yours. 

<img width="1301" alt="Screen Shot 2021-11-22 at 10 24 50 AM" src="https://user-images.githubusercontent.com/18352271/142898148-34747834-bd1f-488b-8945-be2ce90f062a.png">.
